pymeasure/instruments/anritsu/anritsuMS2830A.py

```diff
@@ -23,6 +23,30 @@
 #
 
 from pymeasure.instruments.spectrum_analyzer import SpectrumAnalyzer
+def anritsu_get_trace_mode(mode):
+    mode_map = {
+        'WRIT;OFF': 'WRIT',
+        'WRIT;MINH': 'MINH',
+        'WRIT;MAXH': 'MAXH',
+        'BLAN;OFF': 'BLANK',
+        'BLAN;MINH': 'BLANK',
+        'BLAN;MAXH': 'BLANK',
+        'VIEW;OFF': 'VIEW',
+        'VIEW;MINH': 'VIEW',
+        'VIEW;MAXH': 'VIEW',
+    }
+    return mode_map[mode]
+    
```

Comment:
Combinations *;LAV o *;AVER are not taken into account and probably give an error.
Or I miss something ?

Reply:
Ok, thanks